src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java

```diff
@@ -43,10 +45,19 @@ public void deacquire() {
     }
 
     public void flipUp() {
-        acquirerSolenoid.set(true);
+        acquirerFlipSolenoid.set(true);
     }
 
     public void flipDown() {
-        acquirerSolenoid.set(false);
+        acquirerFlipSolenoid.set(false);
+        Timer.delay(0.5);
```

Comment:
I'd suggest removing the delay.

Subsystems should never have time delays in their methods. That will make it hard to predict and control the code, and it may freeze the bots inputs. 

If you want to make a delay between two actions, use a Command Group.